scripts/rename_galois_to_katana.sh

```diff
@@ -190,74 +188,76 @@ fi
 if [[ ${MODE} == "files" ]]; then
   ### Include directories
   for f in lib*/include/galois; do
-    if [[ ! -e ${f} ]]; then
+    if [[ ! -e "$f" ]]; then
       # Glob failed
       continue
     fi
     # use rsync so we get consistent behavior even if target directory exists
-    rsync -a --remove-source-files ${f}/. $(dirname ${f})/katana
+    targetdir=$(dirname "$f")
```

Comment:
If you want to be space safe you need to quote the `$(...)` too. This issue appears several times.